build/Dockerfile_webapp

```diff
@@ -1,4 +1,4 @@
-FROM node:8.11.3-alpine
+FROM node:10.15.1-alpine
 
 ARG WEBAPP_NAME
 ARG WEBAPP_PORT
```

Comment:
We can kill this file, but maybe we remove all the `./build` stuff in a separate PR.